TurkishFilter/sections/specific.txt

```diff
@@ -99,6 +99,9 @@ spyhackerz.org##.p-body > .p-body-inner > div[style="text-align: center"] > a.li
 ||i.ibb.co/*/kitap-banner.gif$domain=spyhackerz.org
 ||fullhdfilmcibaba.com/video_ob*.js
 ||fullhdfilmcibaba.com/main_tabs*.js
+kriptokoin.com##.header > div.row > div[style="max-height:130px"]
+kriptokoin.com##img[data-image-title="realbetis"]
+kriptokoin.com###text-157
```

Comment:
Better rewrite these rules. Also you need to use ext-css for blocking a leftover.